### PR TITLE
Improvements: Bigger PR and we got docstrings!

### LINKED ISSUE #134: Docstring not rendering in documentation
### Summary

Something wrong with Sphinx ig?

### Reproduction Steps

nothing. This is a problem for sphinx

### Minimal Reproducible Code

_No response_

### Expected Results

Docstrings to render

### Actual Results

They dont

### Intents

_No response_

### System Information

ANYYTHING

### Checklist

- [X] This bug was found on the latest version of EpikCord
- [X] You have checked that there is no issue already mentioning this bug, open or closed
- [X] Full details of the bug have been provided/shown, including full tracebacks in case of exceptions
- [X] All bot tokens have been hidden/omitted from any screenshots or code, if any

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by nav-github01001**
## Description

This is a bigger PR than #154 it is the same except a few stuff
## Checklist

- [x] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [x] - This PR fixes an already existing issue
- [x] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [x] - This PR fixes #134 a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [x] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

**Review by TheUntraceable — APPROVED** [review]

**Review by Sigmanificient — APPROVED** [review]

File: examples/event.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Registering Events
+
+from EpikCord import Client, Intents, Messageable
+
+intents = Intents().all()
```
Comment (ZenoABC) [inline examples/event.py:5]: This is supposed to be `Intents.all()`
Unless ya'll changed intents code, but last time i changed it was to inherit my Flag class.

**Review by ZenoABC — APPROVED** [review]
Looks good to me.

**Review by TheUntraceable — APPROVED** [review]

**Comment by TheUntraceable** [comment]
Shall I merge?

**Comment by nav-github01001** [comment]
Merge when

**Review by Sigmanificient — APPROVED** [review]

**Comment by TheUntraceable** [comment]
> Shall I merge?
I asked you. You didn't reply

File: examples/event.py (hunk 1 above)
Comment (TheUntraceable) [reply examples/event.py:5]: ???

**Comment by nav-github01001** [comment]
Yes do it now
But after remember to change the intents() to intents

**Review by TheUntraceable — APPROVED** [review]

**Pull request merged**